lib/gameplay.rb

```diff
@@ -61,22 +63,20 @@ def board_setup
     user_place_cruiser
   end
 
-  def computer_place_ships
-    cruiser_coordinates = []
-    # require 'pry'; binding.pry
-    until @computer_board.valid_placement?(@computer_cruiser, cruiser_coordinates) do
-      random_coords = @computer_board.cells.keys.sample(3)
-      # require 'pry'; binding.pry
-      cruiser_coordinates << random_coords
-    end
-    @computer_board.place(@computer_cruiser, cruiser_coordinates)
 
-    sub_coordinates = []
-    until @computer_board.valid_placement?(@computer_sub, sub_coordinates) do
-      sample = @computer_board.cells.keys.sample(2)
-      sub_coordinates << sample
+  def computer_place_ship(ship)
+    loop do
```

Comment:
I like the update of having a parameter to pass through! much DRYer. Also the loop and times methods make a lot of sense here. Good job figuring that out!